plugins/widget/plugin.js

```diff
@@ -3890,7 +3890,8 @@
  * comma-separated list of upcast methods from the {@link #upcasts} object.
  *
  * The upcast function **is not** executed in the widget context (because the widget
- * does not exist yet) and two arguments are passed:
+ * does not exist yet), however it is executed in the widget's definition context.
```

Comment:
In case it goes in, could we make widget's definition a link to the widgetDefinition type? So sth like `{@link CKEDITOR.plugins.widget#definition widget's definition}`?

Is it OK to ask the Reviewer to add it during the review?